### PR TITLE
BlackSky branding round 1.

### PR DESCRIPTION
**Pull request by owbp**
This replaces the tab icon (favicon), removes ESRI copyright/logo at bottom of map, and attempts to load script/css files locally that were previously loaded from arcgis servers.

### CODE REVIEW

File: src/app/storymaps/tpl/core/MainView.css [hunk 1]
```diff
@@ -61,4 +61,8 @@ body.layout-float-left:not(.mobile-view) .esriAttribution {
 body.layout-float-left:not(.mobile-view) .esriControlsBR .logo-med {
 	float: left;
 	margin-top: -14px;
-}
\ No newline at end of file
+}
+
+.map .logo-med {
+	display: none;
```
Comment (owaaa) [inline src/app/storymaps/tpl/core/MainView.css]: I don't think we can do this unless we change basemaps

**Comment by owaaa** [comment]
one icon is pretty huge, we may want to optimize it later

File: src/app/storymaps/tpl/core/MainView.css (hunk 1 above)
Comment (owbp) [inline src/app/storymaps/tpl/core/MainView.css]: We're using OSM, not theirs. They stick that at the bottom of the map for all of them.

**Comment by owaaa** [comment]
lgtm

**Pull request merged**